Assets/Scripts/Menu/SettingsManager.cs

```diff
@@ -0,0 +1,24 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class SettingsManager : MonoBehaviour
+{
+    public static SettingsManager Instance { get; private set; }
+
+    public float Sensitivity { get; set; } = 5f;
```

Comment:
You can just make these variables static and refer to them statically instead of doing the singleton